server/src/test/java/com/feeder/server/controllers/HelloControllerTests.java

```diff
@@ -0,0 +1,27 @@
+package com.feeder.server.controllers;
+
+import com.feeder.server.models.Hello;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
```

Comment:
Recommend add annotation
```suggestion

@SpringBootTest
```